api/lib/infrastructure/serializers/csv/csv-serializer.js

```diff
@@ -76,6 +78,32 @@ function getDataFromColumnNames({ csvLineKeys, headers, line }) {
   return data;
 }
 
+function _extractComplementaryCertificationLabelsFromLine(line) {
+  const COMPLEMENTARY_CERTIFICATION_TRUE_VALUE = 'OUI';
+
+  const complementaryCertificationLabels = [];
+
+  Object.keys(line).map((header) => {
+    if (header.endsWith(COMPLEMENTARY_CERTIFICATION_SUFFIX)) {
```

Comment:
Pour plus de comprehension
```suggestion
    if (isComplementaryCertificationHeader(header)) {
```